### ISSUE TITLE
Create an endpoint for special cases

### ISSUE THREAD

**Issue by jpmurray** [jpmurray/canada-sale-tax-api#7]
Quick example:

- return type of goods/services that are exempted for a rate;
- return type of goods/services that have special extra rates (ex.: Manitoba has a 5% tax on hotel room fees)

**Comment by moga111** [comment]
Hello @jpmurray 

Thanks for the API. can you please add support for 

- Retail Sales Tax
- Quebec Sales Tax

Cheers

**Comment by jpmurray** [comment]
@moga111 You can get provinciale rates with the province endpoint by calling it like so : `https://api.salestaxapi.ca/v2/province/qc`.

I'm not sure I get what you mean by retail sales tax though? Anything missing from the endpoints [in the documentation](https://salestaxapi.ca/)?

**Comment by moga111** [comment]
@jpmurray The api provides great info regarding the pst, hst, and gst, but for QC case for example, they have qst, MB they have rst.

Here is a link:
https://en.wikipedia.org/wiki/Sales_taxes_in_Canada#Provincial_sales_taxes

Thanks!

**Comment by jpmurray** [comment]
@moga111 yes, but it's only a matter of terminology. If you call `/v2/province/qc`, you get:

```json
{
  "start": "2013-01-01 00:00:00",
  "type": "gst,pst",
  "pst": 0.09975,
  "hst": 0,
  "gst": 0.05,
  "applicable": 0.14975,
  "source": "Wikipedia (https://en.wikipedia.org/wiki/Sales_taxes_in_Canada), accessed May 31 2019.",
  "updated_at": "2019-06-01 14:39:07",
  "incoming_changes": false
}
```

In this cast, `pst` is the effective provincial tax for quebec, which is called `qst`. I chose to call the provincial taxes `pst` so I don't get lost in different terminology. Same would go for Manitoba, if you take the RST, it is a provincial tax (PST), they just name it differently.

**Comment by moga111** [comment]
@jpmurray Thanks for the reply and sorry for not following up early.
Yes that clarifies it, and thanks again for the API

**Comment by jpmurray** [comment]
@moga111 Sure thing!

**Comment by jpmurray** [comment]
@moga111 I don't know if you are still using the API, but since it's the only way I can make sure to notify people: some endpoints will be deprecated in favor of the latest version. [Please see the documentation](https://salestaxapi.ca/) !

**Comment by moga111** [comment]
Hello @jpmurray , thanks for the heads-up. I will keep that in mind when I update my app.
